pkgs/tools/security/sshuttle/default.nix

```diff
@@ -10,7 +10,7 @@ python3Packages.buildPythonApplication rec {
     url = "mirror://pypi/s/sshuttle/${name}.tar.gz";
   };
 
-  patches = [ ./sudo.patch ];
+  patches = [ ./sudo.patch ./darwin.patch ];
```

Comment:
that patch is pretty large, could you use `fetchpatch` for it

Reply:
done